src/main/java/eu/tornplayground/tornapi/models/faction/FactionBasic.java

```diff
@@ -0,0 +1,120 @@
+package eu.tornplayground.tornapi.models.faction;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import eu.tornplayground.tornapi.models.Model;
+import eu.tornplayground.tornapi.models.converters.DayDurationConverter;
+import eu.tornplayground.tornapi.models.faction.basic.Member;
+import eu.tornplayground.tornapi.models.faction.basic.Rank;
+
+import java.time.Duration;
+import java.util.Map;
+import java.util.Objects;
+
+public class FactionBasic extends Model {
+    @JsonProperty("ID")
+    private int id;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("tag")
+    private String tag;
+
+    @JsonProperty("tag_image")
+    private String tagImage;
+
+    @JsonProperty("leader")
+    private int leader;
+
+    @JsonProperty("co-leader")
+    private int coLeader;
+
+    @JsonProperty("respect")
+    private int respect;
+
+    @JsonProperty("age")
+    @JsonDeserialize(converter = DayDurationConverter.class)
+    private Duration age;
+
+    @JsonProperty("capacity")
+    private int capacity;
+
+    @JsonProperty("best_chain")
+    private int bestChain;
+
+    // TODO: ranked_wars - i literally don't know what those fields even look like...
+    // TODO: territory_wars - i literally don't know what those fields even look like...
+    // TODO: raid_wars - i literally don't know what those fields even look like...
+    // TODO: peace - i literally don't know what those fields even look like...
```

Comment:
No idea on how to get an example on this one, so [unofficial doc](https://tornapi.tornplayground.eu/faction/attacks#attack_result) it is.

Reply:
I totally forgot that the unofficial documentation exists.. gimme a minute, gonna make those changes

Reply:
added it to the class and the tests